examples/serdes-simple/serdes_simple.md

```diff
@@ -1,6 +1,6 @@
 # SerDes - Serialization/Deserialization
 
-The `serdes` module helps to serialize and deserialize subtypes of Ballerina `anydata`. This example demonstrates how to serialize and deserialize a user defined `record` type.
+The `serdes` module helps to serialize and deserialize the subtypes of Ballerina `anydata`. The `serdes:Proto3Schema` object takes `typedesc` as an argument when instantiating, and maps the given type to a Protocol Buffer schema. The `serialize` and `deserialize` methods of the `serdes:Proto3Schema` object serialize and deserialize data using the generated Protocol Buffer schema. The `serialize` method takes a value of the given type as an argument whereas the `deserialize` method takes `byte[]` as an argument. This example demonstrates how to serialize and deserialize a user-defined `Student` `record` type.
```

Comment:
I think we should mention the type-binding/type-inferring here. Otherwise, it might seem that the `serialize` method requires a type parameter.

Reply:
The `deserialize` method uses type binding. Will update the PR

Reply:
Added with [93e62fd](https://github.com/ballerina-platform/ballerina-distribution/pull/3784/commits/93e62fd7cb779ae33c93b1d0558fd38b4e438735)